scss/forms/_form-range.scss

```diff
@@ -43,10 +43,19 @@
     @include transition($form-range-thumb-transition);
     appearance: none;
 
+    // Boosted mod
+    &:hover {
+      border: $form-range-thumb-border;
```

Comment:
IMO, this line is useless. Keep the file up to date if your fine with this change!

Reply:
Yes. But this needs to fix the focus management (which is done in this PR then) to avoid the border to become orange after the active state.